### PR TITLE
renamed sciencebeam commit to tag

### PR DESCRIPTION
**Pull request by de-code**
for https://github.com/elifesciences/sciencebeam/pull/84

### CODE REVIEW

**Comment by de-code** [comment]
I think I am not quite sure how that works with the formula, where it's already called `SCIENCEBEAM_TAG`.

**Comment by giorgiosironi** [comment]
> not quite sure how that works with the formula

The formula doesn't use these files at all, it does not checkout the repository but only uses the image. I'll check if there is a separate change there.

File: docker-compose.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ services:
         depends_on:
             - sciencebeam
     sciencebeam:
-        image: elifesciences/sciencebeam:${SCIENCEBEAM_COMMIT}
+        image: elifesciences/sciencebeam:${SCIENCEBEAM_TAG}
```
Comment (giorgiosironi) [inline docker-compose.yml:25]: also https://github.com/elifesciences/sciencebeam-texture/pull/28/files#diff-4e5e90c6228fd48698d074241c2ba760R17 needs to change. The tag of the `elifesciences/sciencebeam` image is passed to the build process of the `elifesciences/sciencebeam_texture` image, where it is stored as the label. The formula takes out this label on deploy to choose which sciencebeam to use; the formula itself then seems not to need changes as long as the value is still a valid Docker image tag.

Comment (de-code) [reply docker-compose.yml:25]: Okay, that makes sense now.

**Review by giorgiosironi — APPROVED** [review]

**Pull request merged**